run_pacta_data_preparation.R

```diff
@@ -890,6 +890,38 @@ for (pkg_name in pacta_packages) {
   )
 }
 
+# Create archive files
+if (config[["export_archives"]]) {
+  logger::log_info("Exporting input and output archives.")
+
+  logger::log_debug("Creating inputs zip file.")
+  inputs_zip_file_path <- paste0(data_prep_outputs_path, "_inputs.zip")
+  logger::log_trace("Zip file path: \"{inputs_zip_file_path}\".")
+  zip(
+    zipfile = inputs_zip_file_path,
+    files = unlist(parameters[["input_filepaths"]]),
```

Comment:
```suggestion
    files = normalizePath(unlist(parameters[["input_filepaths"]])),
```

zip will fail if these paths are not absolute and canonical